site/content/in-dev/unreleased/metastores.md

```diff
@@ -22,17 +22,10 @@ type: docs
 weight: 700
 ---
 
-This page documents important configurations for connecting to a production database through [EclipseLink](https://eclipse.dev/eclipselink/).
+This page explains how to configure and use Polaris metastores with either the recommended Relational JDBC or the
+deprecated EclipseLink persistence backends.
 
-## Polaris EclipseLink
-
-Polaris includes EclipseLink plugin by default with PostgreSQL driver.
```

Comment:
This info is lost now, should we copy this to the eclipse link i.e the plugin and postgres driver is packaged with eclipse-link ?